gapis/api/cmd_id_group.go

```diff
@@ -602,6 +602,22 @@ func (g *CmdIDGroup) Cluster(maxChildren, maxNeighbours uint64) {
 	}
 }
 
+// Flatten replaces this node's children with its grandchildren.
+func (g *CmdIDGroup) Flatten() {
+	var newSpans Spans
+	for _, s := range g.Spans {
+		switch v := s.(type) {
+		case *SubCmdRoot:
+			newSpans = append(newSpans, v.SubGroup.Spans...)
```

Comment:
Is this correct? It seems v is the type, not a Span (maybe i'm out of depth here). Shouldn't this be ```s.SubGroup.Spans``` ?

Reply:
Got explanation - Go shenanigans! :D